deepspeed/runtime/engine.py

```diff
@@ -160,6 +160,14 @@ def __init__(self,
             steps_per_output=self.steps_per_print(),
             monitor_memory=False)
 
+        if dist.get_rank() == 0:
+            logger.info(
+                f"DeepSpeed Flops Profiler Enabled: {self.flops_profiler_enabled()}")
+
+        if self.flops_profiler_enabled():
+            self.flops_profiler = FlopsProfiler(self.module, self)
+            self.wall_clock_breakdown()
```

Comment:
What does this line do?

Reply:
wall_clock_breakdown is enabled when flops profiler is enabled. this is done elsewhere and this line is a typo.

Reply:
Got it, so line 169 will be removed.